packages/i18n/load-i18n.js

```diff
@@ -38,7 +39,23 @@ const getMomentChunkImport = locale => {
   }
 };
 
-const getLocalizedStringsChunkImport = locale => {
+const getUiKitChunkImport = locale => {
+  const intlLocale = mapLocaleToIntlLocale(locale);
+  switch (intlLocale) {
+    case 'de':
+      return import(/* webpackChunkName: "i18n-ui-kit-locale-de" */ '@commercetools-frontend/ui-kit/i18n/de.json');
```

Comment:
FYI: we just need to remember to update the path to `.../i18n/data/...` once we do the next ui-kit release

Reply:
Yeah it's funny that these things tend to be breaking releases from the view of UIKit.

Reply:
seems like an easy thing to forget 🙈

Reply:
It depends when we are going to release ui-kit :)

We can also revert the folder structure in the ui-kit...

Reply:
I guess we need to wait anyway for the ui-kit release, as we expect the new locales to be there